pkg/kubelet/api/v1alpha1/runtime/api.pb.go

```diff
@@ -971,7 +971,7 @@ type PodSandbox struct {
 	Metadata *PodSandboxMetadata `protobuf:"bytes,2,opt,name=metadata" json:"metadata,omitempty"`
 	// The state of the PodSandbox
 	State *PodSandBoxState `protobuf:"varint,3,opt,name=state,enum=runtime.PodSandBoxState" json:"state,omitempty"`
-	// Creation timestamps of the sandbox
+	// Creation timestamps of the sandbox in nanoseconds
```

Comment:
I wanted to, but because the comments in front and after this line all don't have `.`.